observability/readme.md

```diff
@@ -89,19 +89,39 @@ third-party tools like Grafana or Power BI Dashboards.
 ## Recommended Practices
 
 1. **Correlation Id**: Include unique identifier at the start of the interaction to tie down aggregated data from various system components and provide a holistic view. Read more guidelines about using [correlation id](correlation-id.md).
-2. Ensure health of the services are **monitored** and provide insights into system's performance and behavior.
-3. Ensure **dependent services** are monitored properly. Errors and exceptions in dependent services like Redis cache, Service bus, etc. should be logged and alerted. Also metrics related to dependent services should be captured and logged.
-4. **Faults, crashes, and failures** are logged as discrete events. This helps engineers identify problem area(s) during failures.
-5. Ensure logging configuration (eg: setting logging to "verbose") can be controlled without code changes.
-6. Ensure that **metrics** around latency and duration are collected and can be aggregated.
-7. Start small and add where there is customer impact. [Avoiding metric fatigue](pitfalls.md#metric-fatigue) is very crucial to collecting actionable data.
-8. It is important that every data that is collected contains relevant and rich context.
-9. Personally Identifiable Information or any other customer sensitive information should never be logged. Special attention should be paid to any local privacy data regulations and collected data must adhere to those. (ex: GPDR)
+1. Ensure health of the services are **monitored** and provide insights into system's performance and behavior.
+1. Ensure **dependent services** are monitored properly. Errors and exceptions in dependent services like Redis cache, Service bus, etc. should be logged and alerted. Also metrics related to dependent services should be captured and logged.
+
+    - Additionally, failures in **dependent services** should be propoagated up each level of the stack by the healthcheck.
+
+1. **Faults, crashes, and failures** are logged as discrete events. This helps engineers identify problem area(s) during failures.
+1. Ensure logging configuration (eg: setting logging to "verbose") can be controlled without code changes.
+1. Ensure that **metrics** around latency and duration are collected and can be aggregated.
+1. Start small and add where there is customer impact. [Avoiding metric fatigue](pitfalls.md#metric-fatigue) is very crucial to collecting actionable data.
+1. It is important that every data that is collected contains relevant and rich context.
+1. Personally Identifiable Information or any other customer sensitive information should never be logged. Special attention should be paid to any local privacy data regulations and collected data must adhere to those. (ex: GPDR)
 
 Read more [here](pitfalls.md) to understand what to watch out for while designing and building an observable system.
 
 ## Recipes
 
+### Service Mesh
```

Comment:
This is a helpful content. My suggestion will be to add this content to tools.md page by updating the title to "Tools and Patterns", as I think it fits better as a separate section

Reply:
done! also added a note to the pillars of observability section, pointing the the new section in tools.md